### PR TITLE
CSS Format/Minify Tool

### PR DESCRIPTION
**Pull request by nosecreek**
## Type of Pull Request

- [ ] Bug fix
- [x] Enhancement

Related Issue #s or links (if any): 
#14 

## Description of Changes
Added a CSS page which includes a formatter and minifier.

Note: minification is not currently available for SCSS/LESS. SCSS cannot currently be run in the browser (see https://github.com/sass/dart-sass/issues/25).

### CODE REVIEW

File: __TESTS__/css.spec.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    await user.clear(css);
+    await user.type(
+      css,
+      'body{background:#ffffff;}'.replace(/[{[]/g, '$&$&'),
+    );
+    expect(formattedCss.innerHTML).toContain('body {');
+    expect(formattedCss.innerHTML).toContain('background: #ffffff;');
+    expect(formattedCss.innerHTML).toContain('\n');
+  });
+
```
Comment (dlford) [inline __TESTS__/css.spec.tsx:26]: Can we also test that minify works correctly?

File: __TESTS__/css.spec.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    const css = screen.getByLabelText(/^CSS$/i);
+    const formattedCss = screen.getByLabelText(/FormattedCSS/i);
+
+    await user.clear(css);
+    await user.type(
+      css,
+      'body{background:#ffffff;}'.replace(/[{[]/g, '$&$&'),
+    );
+    expect(formattedCss.innerHTML).toContain('body {');
```
Comment (dlford) [inline __TESTS__/css.spec.tsx:22]: I'd rather see us match exact output here, like the JSON tests.

**Review by dlford — CHANGES_REQUESTED** [review]
@nosecreek Great job on this! I just have a couple notes if it's not too much trouble.

**Comment by nosecreek** [comment]
I think those are both great suggestions. Should be fixed by the latest commit.

**Comment by dlford** [comment]
@nosecreek Great, thank you!

**Pull request merged**